client/src/components/SingleVideo.js

```diff
@@ -0,0 +1,63 @@
+import { useState } from "react";
+import { FaThumbsUp, FaThumbsDown } from "react-icons/fa";
+import "./SingleVideo.css";
+
+const SingleVideo = ({ video, handleDeleteVideoClick }) => {
+  const { title, url, rating, id } = video;
+  const [votes, setVotes] = useState(0); // can be 0, 1 or -1 => to add to rating when thumbsUp or ThumbsDown is clicked
+  const iconSize = 30;
+
+  const handleUpVote = () => {
+    setVotes((prev)=>prev+=1);
+  };
+
+  const handleDownVote = () => {
+    setVotes((prev)=>prev-=1);
+  
+  };
+
+  return (
+    <div className="video-card">
+      <h3>{title}</h3>
+      <div className="iframe-container">
+        <iframe
+          className="responsive-iframe"
+          width="560"
+          height="310"
+          src={`https://www.youtube.com/embed/${url.split("v=")[1]}`}
```

Comment:
What would happen if a URL _didn't_ contain `v=`? Is there a way you could handle that better?